src/Microsoft.ML/Models/BinaryClassificationEvaluator.cs

```diff
@@ -23,7 +24,7 @@ public sealed partial class BinaryClassificationEvaluator
         /// <returns>
         /// A BinaryClassificationMetrics instance that describes how well the model performed against the test data.
         /// </returns>
-        public BinaryClassificationMetrics Evaluate(PredictionModel model, ILearningPipelineLoader testData)
+        public List<BinaryClassificationMetrics> Evaluate(PredictionModel model, ILearningPipelineLoader testData)
```

Comment:
>Evaluate [](start = 49, length = 8)

Why would a singe model produce a list of metrics? What does the list index corresponds to?

Reply:
The list makes sense if you have a list of input models, but not if there is only one.

---
In reply to: [190402103](https://github.com/dotnet/machinelearning/pull/212#discussion_r190402103) [](ancestors = 190402103)

Reply:
Correct. My bad.

---
In reply to: [190402722](https://github.com/dotnet/machinelearning/pull/212#discussion_r190402722) [](ancestors = 190402722,190402103)